tla/consensus/MCccfraft.tla

```diff
@@ -90,7 +93,25 @@ MCNotifyCommit(i,j) ==
 MCInMaxSimultaneousCandidates(i) ==
     Cardinality({ s \in GetServerSetForIndex(i, commitIndex[i]) : state[s] = Candidate}) < 1
 
-mc_spec == Spec
+\* Alternative to CCF!InitReconfigurationVars that sets the initial configuration to the first config defined in Configurations
+MCInitReconfigurationVars ==
+    /\ reconfigurationCount = 0
+    /\ removedFromConfiguration = {}
+    /\ configurations = [i \in Servers |-> 0 :> Configurations[1]]
+
+\* Alternative to CCF!Init that uses the above MCInitReconfigurationVars
+MCInit ==
+    /\ MCInitReconfigurationVars
+    /\ CCF!InitMessagesVars
+    /\ CCF!InitServerVars
+    /\ CCF!InitCandidateVars
+    /\ CCF!InitLeaderVars
+    /\ CCF!InitLogVars
+
+\* Alternative to CCF!Spec that uses the above MCInit
+mc_spec ==   
+    /\ MCInit
+    /\ [][Next]_vars
```

Comment:
I tried to use `CCF!Next` instead of `Next` here but it broke the coverage checking, vscode plug-in reported only `MCInit`.

Reply:
Could be a bug in TLC.  How did `CCF!Next` break coverage checking?

Reply:
using `Next`
![image](https://github.com/microsoft/CCF/assets/1835251/5bfe9a0b-be62-45da-af43-01ed9d12044b)
using `CCF!Next`
![image](https://github.com/microsoft/CCF/assets/1835251/10490bd1-7b04-4369-80d5-830e4776f71c)

Reply:
Can you please also post the content of the .out file, i.e. what's behind "Full Output"?